### PR TITLE
First cleanup of the theme files

### PR DESCRIPTION
**Pull request by SimonBrandner**
This pull request adds the colours from Figma and uses them inside the theme files where possible. It should be reviewable commit-by-commit

<hr>

element-web notes: none

### CODE REVIEW

**Comment by SimonBrandner** [comment]
CI fails should be fixed by https://github.com/matrix-org/matrix-react-sdk/pull/6541

**Comment by Palid** [comment]
@janogarcia could you look at that to double-check we got everything right?

File: res/themes/dark/css/_dark.scss [hunk 1]
```diff
@@ -201,17 +217,17 @@ $reaction-row-button-selected-border-color: $accent-color;
...
-$interactive-tooltip-fg-color: #ffffff;
+$interactive-tooltip-fg-color: $primary-content;
 
 $breadcrumb-placeholder-bg-color: #272c35;
 
 $user-tile-hover-bg-color: $header-panel-bg-color;
 
 $message-body-panel-fg-color: $secondary-fg-color;
-$message-body-panel-bg-color: #394049; // "Dark Tile"
+$message-body-panel-bg-color: $quinary-content; // "Dark Tile"
```
Comment (turt2live) [inline res/themes/dark/css/_dark.scss:230]: we can remove the comment

File: res/themes/light/css/_light.scss [hunk 2]
```diff
@@ -194,13 +210,13 @@ $roomtile-selected-bg-color: #FFF;
...
 $presence-away: #d9b072;
-$presence-offline: #E3E8F0;
+$presence-offline: $quinary-content;
 
 // ********************
 
 $username-variant1-color: #368bd6;
 $username-variant2-color: #ac3ba8;
-$username-variant3-color: #0DBD8B;
+$username-variant3-color: $accent;
```
Comment (turt2live) [inline res/themes/light/css/_light.scss:219]: This is an incorrect usage of the variable

File: res/themes/dark/css/_dark.scss [hunk 3]
```diff
@@ -79,7 +96,7 @@ $menu-bg-color: $header-panel-bg-color;
 $menu-box-shadow-color: $bg-color;
 $menu-selected-color: $room-highlight-color;
 
-$avatar-initial-color: #ffffff;
+$avatar-initial-color: $primary-content;
```
Comment (turt2live) [inline res/themes/dark/css/_dark.scss:99]: This is an incorrect usage of the variable

File: res/themes/light/css/_light.scss [hunk 4]
```diff
@@ -252,24 +268,24 @@ $e2e-warning-color: #ba6363;
 
 /*** ImageView ***/
 $lightbox-bg-color: #454545;
-$lightbox-fg-color: #ffffff;
-$lightbox-border-color: #ffffff;
+$lightbox-fg-color: $background;
```
Comment (turt2live) [inline res/themes/light/css/_light.scss:271]: This doesn't feel right: I think we're trying to hardcode the image view styles, not make them variable.

File: res/themes/light/css/_light.scss [hunk 5]
```diff
@@ -252,24 +268,24 @@ $e2e-warning-color: #ba6363;
...
-$tab-label-active-fg-color: #ffffff;
+$tab-label-active-fg-color: $background;
 $tab-label-bg-color: transparent;
 $tab-label-active-bg-color: $accent-color;
 $tab-label-icon-bg-color: #454545;
 $tab-label-active-icon-bg-color: $tab-label-active-fg-color;
 
 // Buttons
-$button-primary-fg-color: #ffffff;
+$button-primary-fg-color: $background;
```
Comment (turt2live) [inline res/themes/light/css/_light.scss:283]: There's just something wrong about setting a foreground colour to a background colour. We should fix the variable naming so it reads less insane throughout.

File: res/themes/light/css/_light.scss [hunk 6]
```diff
@@ -318,17 +334,17 @@ $kbd-border-color: $reaction-row-button-border-color;
...
-$interactive-tooltip-fg-color: #ffffff;
+$interactive-tooltip-fg-color: $background;
 
 $breadcrumb-placeholder-bg-color: #e8eef5;
 
 $user-tile-hover-bg-color: $header-panel-bg-color;
 
 $message-body-panel-fg-color: $secondary-fg-color;
-$message-body-panel-bg-color: #E3E8F0; // "Separator"
+$message-body-panel-bg-color: $quinary-content; // "Separator"
```
Comment (turt2live) [inline res/themes/light/css/_light.scss:347]: Comment can go

File: res/themes/light/css/_light.scss [hunk 7]
```diff
@@ -337,7 +353,7 @@ $message-body-panel-icon-bg-color: $system-light;
 $voice-record-stop-symbol-color: #ff4b55;
 $voice-record-live-circle-color: #ff4b55;
 
-$voice-record-stop-border-color: #E3E8F0; // "Separator"
+$voice-record-stop-border-color: $quinary-content; // "Separator"
```
Comment (turt2live) [inline res/themes/light/css/_light.scss:356]: Comment can go

**Review by turt2live — CHANGES_REQUESTED** [review]
overall this is probably fine, but I'm a little concerned about where/how the variables are used.

**Review by turt2live — APPROVED** [review]
Looks good! Out of an abundance of caution, I'm going to block this on the RC happening: it's uncomfortably close to the end of the cycle for such a visual change.

**Pull request merged**